collections/_article/dev-snapshot-godot-4-1-dev-1.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Dev snapshot: Godot 4.1 dev 1"
+excerpt: "Here be dragons! First development snapshot for the upcoming Godot Engine 4.1 is available to early adopters and adventurous types."
+categories: ["pre-release"]
+author: Yuri Sizov
+image: /storage/blog/covers/dev-snapshot-godot-4-1-dev-1.jpg
+image_caption_title: Aether's Edge
+image_caption_description: A game by Ancient Stone Studios
+date: 2023-04-21 16:00:00
+---
+
+As we have [announced earlier this week](/article/release-management-4-1/) Godot development is ramping up and you can expect Godot Engine 4.1 to be stable before mid-Summer this year. Right now we are in the middle of the feature merging phase, with the main focus being on improving usability, performance, and the feature set of the engine based on [your feedback](https://github.com/godotengine/godot/issues) and [our prioritized plans](/article/rendering-priorities-4-1/).
```

Comment:
I'm a bit concerned that the last sentence makes it sound like the focus for everyone is on rendering features. 

It might be better to just say:
```suggestion
As we have [announced earlier this week](/article/release-management-4-1/) Godot development is ramping up and you can expect Godot Engine 4.1 to be stable before mid-Summer this year. Right now we are in the middle of the feature merging phase, with the main focus being on improving usability, performance, and stability.
```

Reply:
We sometimes link to our previous posts in that manner. Give a bit of credit to our readers ;)